content/en/core/releases/4.5.0.md

```diff
@@ -0,0 +1,100 @@
+
+---
+title: "4.5.0 release notes"
+linkTitle: "4.5.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.5.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#8513](https://github.com/medic/cht-core/issues/8513): Align "Target" components with Material design guidelines
+- [#8541](https://github.com/medic/cht-core/issues/8541): Options with long names look wrong in enketo selects
+- [#8686](https://github.com/medic/cht-core/issues/8686): Fix target UI to match Figma
+
+
+## Highlights
+### Schedules can be created by passing array on start_from
+We can now configure start_from to accept array of fields while configuring schedules. This allows users to pass multiple fields and first existing field will be used to create schedules. Read more about schedules in [documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/schedules/#app_settingsjson-schedules). cht-conf 3.20.0 or higher is required to validate config containing arrays of field in start_from. 
+
+### Target components are updated to match material design
+
+We updated target components to match to [material design](https://m3.material.io/components/dialogs/guidelines). Many other components were updated in [4.4.0](https://docs.communityhealthtoolkit.org/core/releases/4.4.0/#uiux-changes).
+
+
+{{< figure src="../images/4.5.0-8513-target-changes" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+### Outbound tasks can be configured on a schedule
+CHT now supports adding cron job on outbound push so that tasks can be run on a schedule based on the cron expression. See more about this [here]({{< relref "apps/reference/app-settings/outbound" >}})
```

Comment:
Link content should explain where you're going. This is helpful if you're just scanning the page rather than reading every word.

```suggestion
The CHT now supports scheduling when outbound push tasks are run based on a cron expression. Read about how to set this up in the [outbound push documentation]({{< relref "apps/reference/app-settings/outbound" >}}).
```